Assets/UXF/Scripts/Etc/UXFDataTable.cs

```diff
@@ -73,6 +73,38 @@ public static UXFDataTable FromCSV(string[] csvLines)
             return table;
         }
 
+        /// <summary>
+        /// Build a table from lines of TSV text.
+        /// </summary>
+        /// <param name="tsvLines"></param>
+        /// <returns></returns>
+        public static UXFDataTable FromTSV(string[] tsvLines)
+        {
+            string[] headers = tsvLines[0].Split('\t');
+            var table = new UXFDataTable(tsvLines.Length - 1, headers);
+
+            // traverse down rows
+            for (int i = 1; i < tsvLines.Length; i++)
+            {
+                string[] values = tsvLines[i].Split('\t');
+
+                // if last line, just 1 item in the row, and it is blank, then ignore it
+                if (i == tsvLines.Length - 1 && values.Length == 1 && values[0].Trim() == string.Empty ) break;
+
+                // check if number of columns is correct
+                if (values.Length != headers.Length) throw new Exception($"TSV line {i} has {values.Length} columns, but expected {headers.Length}");
+
+                // build across the row
+                var row = new UXFDataRow();
+                for (int j = 0; j < values.Length; j++)
+                    row.Add((headers[j], values[j].Trim('\"')));
+
+                table.AddCompleteRow(row);
+            }
+
+            return table;
+        }
```

Comment:
Completely agree :)

Reply:
Also, just changing the experimenter builder is not enough because then what might happen is that a variable such as a string that is to be displayed to the participant contains and a comma, which UXF then tries to save, which doesn't work if the the output file is an .csv. I currently have work-around where I just replace a comma with an underscore before it is saved but tihis prevents me to copy the data from the input file to the result file.